Improve colonies

vincentneko opened the pull request:
This one is long overdue !

Replace text with colonies images when needed

![Screenshot 2020-09-18 18 26 31](https://user-images.githubusercontent.com/56086992/93622062-c2877500-f9dc-11ea-8add-43af3b1cbb61.png)

nwai90 commented:
@vincentneko Do note that the previous text for SP "Build colony (17 MC)" and "Poseidon first action - Select where to build colony" has been replaced:

<img width="406" alt="Screenshot 2020-09-19 at 9 17 08 AM" src="https://user-images.githubusercontent.com/2408094/93655965-6878c500-fa59-11ea-8dad-8a847089a686.png">

<img width="1072" alt="Screenshot 2020-09-19 at 9 15 37 AM" src="https://user-images.githubusercontent.com/2408094/93655963-63b41100-fa59-11ea-8361-43b1ae4fdb23.png">

vincentneko commented:
> @vincentneko Do note that the previous text for SP "Build colony (17 MC)" and "Poseidon first action - Select where to build colony" has been replaced:
> 
> <img alt="Screenshot 2020-09-19 at 9 17 08 AM" width="406" src="https://user-images.githubusercontent.com/2408094/93655965-6878c500-fa59-11ea-8dad-8a847089a686.png">
> 
> <img alt="Screenshot 2020-09-19 at 9 15 37 AM" width="1072" src="https://user-images.githubusercontent.com/2408094/93655963-63b41100-fa59-11ea-8361-43b1ae4fdb23.png">

Good catch ! Will update

nwai90 commented:
Would it make sense to do something similar for Aridor as well? Like `SelectAddColony`

<img width="484" alt="Screenshot 2020-09-19 at 9 14 30 AM" src="https://user-images.githubusercontent.com/2408094/93661411-2caa2380-fa8a-11ea-8314-59828c6a23fe.png">

vincentneko commented:
> Would it make sense to do something similar for Aridor as well? Like `SelectAddColony`
> 
> <img alt="Screenshot 2020-09-19 at 9 14 30 AM" width="484" src="https://user-images.githubusercontent.com/2408094/93661411-2caa2380-fa8a-11ea-8314-59828c6a23fe.png">

Yes, I forgot this one !

nwai90 commented on src/Player.ts:
```diff
@@ -1379,7 +1379,7 @@ export class Player implements ILoadable<SerializedPlayer, Player>{
     private buildColony(game: Game, openColonies: Array<IColony>): PlayerInput {
 
-      let buildColony = new SelectColony("Build colony", "Build", openColonies, (colonyName: ColonyName) => {
+      let buildColony = new SelectColony("Build colony(" + constants.BUILD_COLONY_COST + " MC)", "Build", openColonies, (colonyName: ColonyName) => {
```
@vincentneko Note it's missing a space between `colony` and `(` ; should be `"Build colony (" + constants.BUILD_COLONY_COST + " MC)"` here

vincentneko commented:
@nwai90 Thanks a lot for the great feedback. I've updated regarding your remarks and fixed a couple of things.
Can you please make a final review ?
CSS for the checkmark is not great but can be addressed in another PR I guess.

nwai90 commented:
@vincentneko Just tested, everything looks great!
